### PR TITLE
Fix sass option and nix additional yarn install task

### PR DESCRIPTION
**Pull request by gauravtiwari**
Please see: https://webpack.js.org/loaders/sass-loader/#sassoptions

and since we are supporting Rails > 5.2 we can safely remove webpacker yarn install enhance task.

### CODE REVIEW

File: package/rules/module.sass.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
 module.exports = getStyleRule(/\.(scss|sass)$/i, true, [
   {
     loader: 'sass-loader',
-    options: { sourceMap: true }
+    options: {
+      sourceMap: true,
+      sassOptions: {
+        includePaths
+      }
```
Comment (aried3r) [inline package/rules/module.sass.js]: I believe this means that we should go for sass-loader ^8.0.0, WDYT?

https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#-breaking-changes

Previously:
https://github.com/webpack-contrib/sass-loader/tree/v7.3.1#options
Current:
https://github.com/webpack-contrib/sass-loader/tree/v8.0.2#object

**Comment by aried3r** [comment]
Hey! Thanks for the PR!

Does this solve https://github.com/rails/webpacker/issues/1951 by any chance? As in, does `includePaths` include `.node_modules`?

File: package/rules/module.sass.js (hunk 1 above)
Comment (gauravtiwari) [reply package/rules/module.sass.js]: Ahh I see. Yeah, lets do that. 

I am thinking to try and upgrade master to webpack 5 beta too and see if everything works as intended but in another pr.

**Comment by gauravtiwari** [comment]
Hey, I am not sure really, will work on this pr bit later today and update 👍

**Comment by gauravtiwari** [comment]
closing in favour of #2508 

Will open another pr with remaining changes.

**Pull request closed without merging**